examples/mongodb-atlas/README.md

```diff
@@ -0,0 +1,199 @@
+# MongoDB Atlas
+
+[MongoDB Atlas](https://www.mongodb.com/atlas) is an integrated suite of cloud
+database and data services to accelerate and simplify how you build with data.
+It deploys and scales a MongoDB cluster in the cloud. MongoDB Atlas can be
+primarily classified under "MongoDB Hosting".
+
+## Prerequisites
+
+* Kubernetes 1.20+
+* Kanister controller version 0.90.0 installed in your cluster
+* Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+* MongoDB Atlas account registered with organization, project and cluster
+
+## Integrating with Kanister
+
+In case, if you don't have `Kanister` installed already, use following commands
+to install.
+```bash
+$ helm repo add kanister https://charts.kanister.io
+$ helm install kanister --namespace kanister --create-namespace \
+    kanister/kanister-operator --set image.tag=0.90.0
+```
+
+### Build docker image for MongoDB Atlas
+
+To build docker image for MongoDB Atlas, execute following steps. Image build is
+needed as connection process to MongoDB Atlas is using `mongosh` command. Also,
+it installs `atlas` CLI which is used for performing backup and restore
+operations. These gets installed in the image along with some other required
+packages in blueprint.
```

Comment:
```suggestion
### Build a tools image to interact with MongoDB Atlas

We need to build a tools image that should containe `mongosh`, `atlas` and other related
utilities so that it can be used to interact with the MondoDB Atlas database to add some data to it.
This image will also be used in the Blueprint to run the `atlas` backup and restore
related commands against the MongoDB Atlas database. Please execute the commands
below to build and push the image.
```

Reply:
fixed